### ISSUE TITLE
Forgot to change label during training data agumentation

### ISSUE THREAD

**Issue by addvaluejack** [zllrunning/face-parsing.PyTorch#57]
https://github.com/zllrunning/face-parsing.PyTorch/blob/d2e684cf1588b46145635e8fe7bcc29544e5537e/face_dataset.py#L39

After horizontal fliping an image, we need to change all 'l_eye' labels to 'r_eye', and change all 'r_eye' to 'l_eye', same for 'ear' and 'brow'.
